bifrost_jupyter_extension/bifrost.py

```diff
@@ -80,18 +81,11 @@ def update_graph_from_cols(self, changes):
         # Vega spec is updated from the frontend. To track history, respond to these changes here.
         pass
 
+    @observe("df_code")
+    def update_output_dataframe(self, changes):
+        code = changes["new"]
+        get_ipython().run_cell(code).result
```

Comment:
This causes the issue when there is no variable assigned to store the output of `plot()`